stable/nfs-client-provisioner/templates/clusterrole.yaml

```diff
@@ -0,0 +1,22 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  labels:
+    app: {{ template "nfs-client-provisioner.name" . }}
+    chart: {{ template "nfs-client-provisioner.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "nfs-client-provisioner.fullname" . }}-runner
+rules:
+  - apiGroups: [""]
```

Comment:
should endpoints access be added too? see: https://github.com/kubernetes-incubator/external-storage/blob/master/nfs-client/deploy/auth/clusterrole.yaml

Reply:
Only been added 2 weeks ago so it seems https://github.com/kubernetes-incubator/external-storage/commit/d46083d75be3c046c5313c03c0b4b5f29d9b2ec2. I can sync it though. ;-) Man I really want to get this merged. :-p

Reply:
That would be good - it was malfunctioning for me without it